File: src/main/java/onl/tesseract/command/WorldsCommand.java
package onl.tesseract.command;

import onl.tesseract.commandBuilder.CommandContext;
import onl.tesseract.commandBuilder.annotation.Command;
import onl.tesseract.commandBuilder.annotation.CommandBody;
import onl.tesseract.menu.MenuMenu;
import onl.tesseract.menu.TPWorldMenu;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

@Command
public class WorldsCommand extends CommandContext {
    @CommandBody
    public boolean onCommand(CommandSender sender)
    {
        if (sender instanceof Player player )
        {
            new TPWorldMenu(new MenuMenu()).open(player);
        }

        return true;
    }
}

File: src/main/java/onl/tesseract/command/WorldsCommand.kt
package onl.tesseract.command

import onl.tesseract.commandBuilder.CommandContext
import onl.tesseract.commandBuilder.annotation.Command
import onl.tesseract.commandBuilder.annotation.CommandBody
import onl.tesseract.menu.TPWorldMenu
import org.bukkit.command.CommandSender
import org.bukkit.entity.Player

@Command
class WorldsCommand : CommandContext() {
    @CommandBody
    fun onCommand(sender: CommandSender?): Boolean {
        if (sender is Player) {
            TPWorldMenu().open(sender)
        }

        return true
    }
}
